apps/src/templates/certificates/Congrats.jsx

```diff
@@ -72,97 +59,250 @@ export default function Congrats(props) {
+    '/../../../../shared/images/courses/logo_hardware.jpg',
+  ];
+
+  const professionalLearning = [
+    {
+      title: i18n.teachWithCodeOrg(),
+      description: i18n.teachWithCodeOrgDescription(),
+      buttonText: i18n.teachWithCodeOrg(),
+      image: 'https://code.org/images/teach-page-top.png',
+      link: 'https://code.org/teach',
+    },
+    {
+      title: i18n.courseOfferingSelfPacedPl(),
+      description: i18n.selfPacedPlDescription(),
+      buttonText: i18n.exploreProfessionalLearning(),
+      image: 'https://code.org/shared/images/banners/self-paced-pl-hero.png',
+      link: 'https://code.org/educate/professional-development-online',
+    },
+  ];
 
   return (
-    <div style={styles.container}>
-      <Certificate
-        tutorial={tutorial}
-        certificateId={certificateId}
-        randomDonorTwitter={randomDonorTwitter}
-        randomDonorName={randomDonorName}
-        under13={under13}
-        initialCertificateImageUrl={initialCertificateImageUrl}
-        isHocTutorial={isHocTutorial}
-      >
-        {renderExtraCertificateLinks(language, tutorial)}
-      </Certificate>
-      {userType === 'teacher' && isEnglish && <TeachersBeyondHoc />}
-      {isHocTutorial && (
-        <StudentsBeyondHoc
-          completedTutorialType={tutorialType}
-          MCShareLink={MCShareLink}
-          userType={userType}
+    <div className={style.wrapper}>
+      <div className={style.certificateContainer}>
+        <Certificate
+          tutorial={tutorial}
+          certificateId={certificateId}
+          randomDonorTwitter={randomDonorTwitter}
+          randomDonorName={randomDonorName}
           under13={under13}
-          isEnglish={isEnglish}
-          hideDancePartyFollowUp={hideDancePartyFollowUp}
-        />
-      )}
-      {!isHocTutorial && (
-        <GraduateToNextLevel
-          scriptName={nextCourseScriptName}
-          courseTitle={nextCourseTitle}
-          courseDesc={nextCourseDesc}
-        />
+          initialCertificateImageUrl={initialCertificateImageUrl}
+          isHocTutorial={isHocTutorial}
+        >
+          {renderExtraCertificateLinks(language, tutorial)}
+        </Certificate>
+      </div>
+
+      <div className={style.continueBeyond}>
+        <Heading3 className={style.textCenter}>
+          {i18n.continueBeyondHourOfCode()}
+        </Heading3>
+        <div
+          className={`${style.actionBlockWrapper} ${style.actionBlockWrapperThreeCol} ${style.courseContainer}`}
+        >
+          {curriculaData.map((item, index) => (
+            <div
+              className={`${style.actionBlock} ${style.actionBlockOneCol} ${style.flexSpaceBetween}`}
+              key={index}
+            >
+              <div className={style.contentWrapper}>
+                <p className={style.overline}>{item.grade}</p>
+                <Heading3>{item.title}</Heading3>
+                <img src={item.image} alt="" />
+                <p>{item.description}</p>
```

Comment:
Similar to using the `Heading3` tags, can you also use `BodyOneText` (or `BodyTwoText`, `BodyThreeText`, etc. if one of those looks more right visually) for `<p>` tags in here